src/content/docs/infrastructure/host-integrations/host-integrations-list/postfix-monitoring-integration.mdx

```diff
@@ -376,28 +356,27 @@ You can choose our pre-built dashboard template named `Postfix` to monitor your
 
 Your Postfix dashboard is considered a custom dashboard and can be found in the <DoNotTranslate>**Dashboards**</DoNotTranslate> UI. For docs on using and editing dashboards, see [our dashboard docs](/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards).
```

Comment:
I would spruce this step up and make it a 'View your data' pattern per other OHIs. I also don't think we should err on the side of linking out to docs in branching oHI steps bc it gets tedious + might deter customers from getting any value from it